### DISCUSSION TITLE
POrt 25, ipv4 and ipv6

### DISCUSSION THREAD

**Discussion by heavestudio** [postalserver/postal#3118] [Help with using Postal]
## Describe the bug

While testing portal as smtp for a wordpress install, it first worked.
Then i tried in another wordpress install and it gave me the can't connect to server.

as i tried to understand why, it seems that on ipv4 port 25 isn't listened

@Postal:/# sudo netstat -tuln | grep :25
tcp6       0      0 :::25                   :::*                    LISTEN

as anyone encoutered the same situation ?

**Comment by willpower232** [comment]
If something is listening on the IPv6 port, it will be listening on the IPv4 port as well.

You will have to use something like `postal logs smtp` to see what your wordpress is doing.

**Reply by heavestudio** [reply]
postal logs smtp

list somet stuff but doesn't seems to be related to what wordpress is doing, (timestamp doesn't matches)

**Reply by willpower232** [reply]
Unless you can retry wordpress and have another look at the logs, you'll have to see what logs are in wordpress then, I'm not sure what information you will see.

It is also possible that the time in the logs is UTC which may not match where you are currently.

**Reply by heavestudio** [reply]
Hi Will, 

So after trying, again i can see no additional logs in with postal logs smtp, it like it never reached the smtp.

On nginx logs i can see that fluent stmp get  422 errors,

**Reply by heavestudio** [reply]
False alert, in fact the wordpress host blocked outgoing trafic from port 25
